packages/backend/src/engine/amplify_stack.ts

```diff
@@ -93,3 +108,12 @@ class CognitoRoleTrustPolicyValidator implements IAspect {
     }
   };
 }
+
+// This aspect sets removal policy of all resources to destroy for sandbox deployments
+class SandboxRemovalPolicyDestroyAspect implements IAspect {
+  visit(node: IConstruct): void {
+    if (CfnResource.isCfnResource(node)) {
+      node.applyRemovalPolicy(RemovalPolicy.DESTROY);
+    }
+  }
```

Comment:
an edge case to consider.

should we refrain from this if customer set value explicitly in their code? (can we detect this?).

alternatively should we warn that we're overriding ?

Reply:
We can check the value but I'm not sure if we can detect if it was explicitly set, let me look into this.